listenbrainz/tests/integration/test_stats_api.py

```diff
@@ -24,14 +24,43 @@ def test_artist(self):
         self.assert200(response)
         data = json.loads(response.data)['payload']
         sent_count = payload['count']
-        recieved_count = data['all_time']['count']
+        recieved_count = data['count']
```

Comment:
It's only a variable name in code, so not super important, but it's "receive", not "recieve"